python/lsst/ts/rubintv/main.py

```diff
@@ -120,10 +119,13 @@ async def add_cache_control_header(
 
     # Attach the routers.
     app.include_router(internal_router)
-    app.include_router(ddv_ws_router, prefix=f"{config.path_prefix}/ddv")
+    app.include_router(internal_ws_router, prefix="/ws")
+    app.include_router(data_ws_router, prefix=f"{config.path_prefix}/ws/data")
+    app.include_router(
+        heartbeat_ws_router, prefix=f"{config.path_prefix}/ws/heartbeats"
+    )
+    app.include_router(ddv_client_ws_router, prefix=f"{config.path_prefix}/ws/ddv")
```

Comment:
Wouldn't it be better to have `data_ws_router`, `heartbeat_ws_router` and `ddv_client_ws_router` inside a `external_ws_router` (as all share the `f"{config.path_prefix}/ws/` prefix)? You could then have the different paths for `data`, `heartbeats` and `ddv` on the same handler and a different one for the internal ws one.

Reply:
I like the separation of concerns in the files as they are. Maybe as a compromise, I could define:
`external_ws_router_url = prefix=f"{config.path_prefix}/ws"`
and then use that in the router defs?